performance/static/js/eqprs-file-upload.js

```diff
@@ -0,0 +1,19 @@
+$(document).ready(function () {
+    prepareUIElements();
+});
+
+function prepareUIElements() {
+    document.getElementById('eqprsfileupload_form').addEventListener('submit', formSubmit);
+
+    enableDisableElement('id_user', true);
+}
+
+function formSubmit() {
+    enableDisableElement('id_user', false);
+}
+
+function enableDisableElement(id, enable) {
+    if (document.getElementById(id)) {
+        document.getElementById(id).disabled = enable;
+    }
+}
```

Comment:
I think this can be deleted when the creating user is set server-side

Reply:
done